README.md

```diff
@@ -1,31 +1,36 @@
 ## Developer guide
 
 This database project is dependent upon multiple other databases to work correctly. At the time of writing, there is no easy way to restore a local copy of each of these dependencies. Moreover, the project requires the databases to all run on the same instance of SQL Server, meaning that we do not currently have a way to set up a local development environment for this project.
-
-In most cases, development should be done against the `int` environment on the `ntbs-ops-dbs` server. Since it may be difficult to resolve conflicts with multiple developers working against the same database, the currently recommended branching strategy is to commit changes directly onto the `master` branch.
+However, it is possible to set up a new reporting database in this environment, to do your development against in a feature branch. In most cases, the dependencies of the reporting databases should be the `int` versions of the relevant databases (`int-geography`, `int-ntbs`, etc.)
 
 Make sure that you have the following tools installed locally:
 
 - Visual Studio 2017 (or later) - this can be downloaded from https://visualstudio.microsoft.com/downloads/
-	- During installation, select �Data storage and processing�
+	- During installation, select "Data storage and processing"
 
 - Git - this can be downloaded from https://git-scm.com/downloads
 
 You can then begin working on the project by cloning the repository and opening `VisualStudio_ntbs-reporting.sln` in Visual Studio.
 
+To make a new copy of the reporting database, do the following:
+1. Make a copy of `azure-ntbs-int-reporting.publish.xml` and name it `azure-ntbs-USER-reporting.publish.xml`. This file will be ignored by Git.
+2. Edit this file, replacing the `TargetDatabaseName` value with the name of the new reporting database you would like to create. This database should not exist on the Azure SQL Server yet.
+3. Publish this database, by double clicking on the xml file in the VS Solution explorer:
+    1. In the pop-up window, next to the `Target database connection` select `Edit...`, and enter the password for the `sqlAdmin` user - this can be found in the `ntbs-ops-dbs-credentials` secret in Azure. (If you also check `Remember password` then you do not need to repeat this step in future).
+    2. Click `Generate Script`. Review the SQL script that is generated, and confirm that it is reflective of your changes. In this case, it should be creating all of the tables, and adding all of the functions and stored procedures.
+    3. When you are happy with this script, double click on the xml file again and then click `Publish` in the dialog box that pops up. This will generate the same script again, and then run it on the SQL server, creating your new database in the process.
+4. To populate the database, first run `uspPopulateCalendarTable`, and then run `uspGenerate`. This database is now ready for use.
```

Comment:
You also need to run the script which can be found here: https://github.com/publichealthengland/ntbs-reporting/blob/master/source/Scripts/PopulateFeatureFlags.sql  specifying the right values for your environment (for development this will always be 1, 1, 1

Reply:
I think we did do this, but I then forgot about it. Sorry about that! Added in now